[CI] ClusterDisruptionIT testAckedIndexing failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #4916 / rhel-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/wazh3qxsy6b7e)
- [elasticsearch-periodic-platform-support #4881 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/l2ygljkl55lka)

**Reproduction Line:**
```
./gradlew ":server:internalClusterTest" --tests "org.elasticsearch.discovery.ClusterDisruptionIT.testAckedIndexing" -Dtests.seed=5B73A8D843BA4357 -Dtests.locale=da-DK -Dtests.timezone=SystemV/CST6CDT -Druntime.java=23
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.discovery.ClusterDisruptionIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testAckedIndexing),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: timed out waiting for green state
```

**Issue Reasons:**
- [8.x] 2 failures in test testAckedIndexing (0.8% fail rate in 258 executions)
- [8.x] 2 failures in pipeline elasticsearch-periodic-platform-support (25.0% fail rate in 8 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

Concerning for `[elasticsearch-periodic-platform-support #4919 / oraclelinux-7_platform-support-unix](https://gradle-enterprise.elastic.co/s/h3hx2chmhinck)`.

```
  2> java.lang.AssertionError: doc [14] exists [1] times in index
  2> 	at __randomizedtesting.SeedInfo.seed([7E3645A3E1F0C921]:0)
  2> 	at org.elasticsearch.index.engine.InternalEngine.assertDocDoesNotExist(InternalEngine.java:1586)
  2> 	at org.elasticsearch.index.engine.InternalEngine.indexIntoLucene(InternalEngine.java:1415)
  2> 	at org.elasticsearch.index.engine.InternalEngine.index(InternalEngine.java:1219)
  2> 	at org.elasticsearch.index.shard.IndexShard.index(IndexShard.java:1084)
  2> 	at org.elasticsearch.index.shard.IndexShard.applyIndexOperation(IndexShard.java:1010)
  2> 	at org.elasticsearch.index.shard.IndexShard.applyTranslogOperation(IndexShard.java:2006)
  2> 	at org.elasticsearch.index.shard.IndexShard.applyTranslogOperation(IndexShard.java:1993)
  2> 	at org.elasticsearch.indices.recovery.RecoveryTarget.lambda$indexTranslogOperations$4(RecoveryTarget.java:454)
  2> 	at org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:356)
  2> 	at org.elasticsearch.indices.recovery.RecoveryTarget.indexTranslogOperations(RecoveryTarget.java:429)
  2> 	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$TranslogOperationsRequestHandler.performTranslogOps(PeerRecoveryTargetService.java:658)
  2> 	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$TranslogOperationsRequestHandler.handleRequest(PeerRecoveryTargetService.java:605)
  2> 	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$TranslogOperationsRequestHandler.handleRequest(PeerRecoveryTargetService.java:597)
  2> 	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$RecoveryRequestHandler.messageReceived(PeerRecoveryTargetService.java:691)
  2> 	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$RecoveryRequestHandler.messageReceived(PeerRecoveryTargetService.java:678)
  2> 	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:76)
  2> 	at org.elasticsearch.transport.InboundHandler.doHandleRequest(InboundHandler.java:289)
  2> 	at org.elasticsearch.transport.InboundHandler$1.doRun(InboundHandler.java:302)
  2> 	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1023)
  2> 	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
  2> 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
  2> 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
  2> 	at java.base/java.lang.Thread.run(Thread.java:1575)
```

Less concerning error for `[elasticsearch-intake #13071 / part1](https://gradle-enterprise.elastic.co/s/ixl2x4hz6v6sk)`

All the failures are from the assertion on duplicated docs during indexing

```
WARNING: Uncaught exception in thread: Thread[#4779,elasticsearch[node_t2][generic][T#4],5,TGRP-ClusterDisruptionIT]
java.lang.AssertionError: doc [27] exists [1] times in index
	at __randomizedtesting.SeedInfo.seed([226D48BE032B3BBC]:0)
	at org.elasticsearch.index.engine.InternalEngine.assertDocDoesNotExist(InternalEngine.java:1586)
	at org.elasticsearch.index.engine.InternalEngine.indexIntoLucene(InternalEngine.java:1415)
	at org.elasticsearch.index.engine.InternalEngine.index(InternalEngine.java:1219)
	at org.elasticsearch.index.shard.IndexShard.index(IndexShard.java:1084)
	at org.elasticsearch.index.shard.IndexShard.applyIndexOperation(IndexShard.java:1010)
	at org.elasticsearch.index.shard.IndexShard.applyTranslogOperation(IndexShard.java:1990)
	at org.elasticsearch.index.shard.IndexShard.applyTranslogOperation(IndexShard.java:1977)
	at org.elasticsearch.indices.recovery.RecoveryTarget.lambda$indexTranslogOperations$4(RecoveryTarget.java:454)
	at org.elasticsearch.action.ActionListener.completeWith(ActionListener.java:356)
	at org.elasticsearch.indices.recovery.RecoveryTarget.indexTranslogOperations(RecoveryTarget.java:429)
	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$TranslogOperationsRequestHandler.performTranslogOps(PeerRecoveryTargetService.java:658)
	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$TranslogOperationsRequestHandler.handleRequest(PeerRecoveryTargetService.java:605)
	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$TranslogOperationsRequestHandler.handleRequest(PeerRecoveryTargetService.java:597)
	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$RecoveryRequestHandler.messageReceived(PeerRecoveryTargetService.java:691)
	at org.elasticsearch.indices.recovery.PeerRecoveryTargetService$RecoveryRequestHandler.messageReceived(PeerRecoveryTargetService.java:678)
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:76)
	at org.elasticsearch.transport.InboundHandler.doHandleRequest(InboundHandler.java:289)
	at org.elasticsearch.transport.InboundHandler$1.doRun(InboundHandler.java:302)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1023)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
```

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -192,15 +192,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.plugin.CanMatchIT
   method: testAliasFilters
   issue: https://github.com/elastic/elasticsearch/issues/134512
-- class: org.elasticsearch.oldrepos.OldRepositoryAccessIT
-  method: testOldRepoAccess
-  issue: https://github.com/elastic/elasticsearch/issues/120148
-- class: org.elasticsearch.oldrepos.OldRepositoryAccessIT
-  method: testOldSourceOnlyRepoAccess
-  issue: https://github.com/elastic/elasticsearch/issues/134646
-- class: org.elasticsearch.oldrepos.OldMappingsIT
-  method: testStandardTokenFilter
-  issue: https://github.com/elastic/elasticsearch/issues/134647
 - class: org.elasticsearch.aggregations.bucket.AggregationReductionCircuitBreakingIT
   method: testCBTrippingOnReduction
   issue: https://github.com/elastic/elasticsearch/issues/134667
@@ -213,9 +204,6 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=transform/transforms_start_stop/Test stop transform with force and wait_for_checkpoint true}
   issue: https://github.com/elastic/elasticsearch/issues/135135
-- class: org.elasticsearch.discovery.ClusterDisruptionIT
-  method: testAckedIndexing
-  issue: https://github.com/elastic/elasticsearch/issues/117024
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=analytics/nested_top_metrics_sort/terms order by top metrics numeric not null double values}
   issue: https://github.com/elastic/elasticsearch/issues/135159
@@ -228,9 +216,6 @@ tests:
 - class: org.elasticsearch.xpack.security.authz.microsoft.MicrosoftGraphAuthzPluginIT
   method: testConcurrentAuthentication
   issue: https://github.com/elastic/elasticsearch/issues/135777
-- class: org.elasticsearch.versioning.ConcurrentSeqNoVersioningIT
-  method: testSeqNoCASLinearizability
-  issue: https://github.com/elastic/elasticsearch/issues/117249
 - class: org.elasticsearch.test.rest.yaml.CcsCommonYamlTestSuiteIT
   method: test {p0=field_caps/10_basic/Field caps for number field with only doc values}
   issue: https://github.com/elastic/elasticsearch/issues/136244

